### PR TITLE
[Network]add --internet-security for express-route gateway connection

### PR DESCRIPTION
**Pull request by MyronFanQiu**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
@haroldrandom  please take a look.

File: src/azure-cli/azure/cli/command_modules/network/_params.py [hunk 1]
```diff
@@ -598,6 +598,7 @@ def load_arguments(self, _):
         c.argument('connection_name', options_list=['--name', '-n'], help='ExpressRoute connection name.', id_part='child_name_1')
         c.argument('routing_weight', help='Routing weight associated with the connection.', type=int)
         c.argument('authorization_key', help='Authorization key to establish the connection.')
+        c.argument('enable_internet_security', options_list='--internet-security', arg_type=get_three_state_flag(), help='Enable internet security.', min_api='2019-09-01')
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/network/_params.py]: I suggest --enable-internet-security

Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/network/_params.py]: The help message, which is same as parameter name, tells nothing-_-||

Comment (haroldrandom) [reply src/azure-cli/azure/cli/command_modules/network/_params.py]: some of the arguments are like --internet-security, which help message is Enable/Disable internet security.

**Comment by MyronFanQiu** [comment]
> 1. Add test.
> 2. Add history entry.

added history. Cannot add test. Manually test it and also asked service team to test it.

File: src/azure-cli/azure/cli/command_modules/network/_params.py (hunk 1 above)
Comment (MyronFanQiu) [reply src/azure-cli/azure/cli/command_modules/network/_params.py]: already have --internet-security in other commands and service team insist on such name.

Comment (MyronFanQiu) [reply src/azure-cli/azure/cli/command_modules/network/_params.py]: asking for help information right now.

**Pull request merged**